src/pybind/docling_parser.h

```diff
@@ -3,7 +3,7 @@
 #ifndef PYBIND_PDF_PARSER_H
 #define PYBIND_PDF_PARSER_H
 
-#include <utf8/utf8.h>
+#include <utf8.h>
```

Comment:
note: this is needed to aligned the include path of the manual externals with the system packages which have `utf8.h` directly in `/usr/include/utf8.h`.